getting-started/src/main/kotlin/com/google/cloud/kotlin/demo/Application.kt

```diff
@@ -0,0 +1,15 @@
+// See https://github.com/JetBrains/kotlin-examples/blob/master/LICENSE
```

Comment:
Let's replace this license with the same Google copyright license in `EmojifyController.kt`